packages/blocks/src/api/parser.js

```diff
@@ -251,6 +273,11 @@ export function getBlockAttribute( attributeKey, attributeSchema, innerHTML, com
 		case 'tag':
 			value = parseWithAttributeSchema( innerHTML, attributeSchema );
 			break;
+
+		default:
+			if ( has( parseOptions.sources, attributeSchema.source ) ) {
```

Comment:
Would it make sense to allow an async retrieval of the value? Imagining cases where external requests are required to retrieve the value. I guess it would increase the overall complexity of the module.